Sources/StreamChat/Models/Attachments/AnyAttachmentPayload.swift

```diff
@@ -4,17 +4,35 @@
 
 import Foundation
 
+/// A protocol an attachment payload type has to conform in order it can be
+/// attached to/exposed on the message.
 public protocol AttachmentPayload: Codable {
+    /// A type of resulting attachment.
     static var type: AttachmentType { get }
 }
 
+/// A type which wraps either a local file URL that has to be uploaded
```

Comment:
-> `A type-erased type that wraps either ...`

Reply:
✅ Updated